Plutarch/Lift.hs

```diff
@@ -70,7 +84,8 @@ The Haskell type is determined by `PLifted p`.
 
 This typeclass is closely tied with 'PConstant'.
 -}
-type PLift = PUnsafeLiftDecl
+type PLift :: PType -> Constraint
+type PLift p = PUnsafeLiftDecl p
```

Comment:
You don't need this change anymore. You can keep the kind signature though I suppose.

Reply:
I like standalone type signatures :)
Removed the change in definition.